sale_order_lot_selection/models/sale_order_line.py

```diff
@@ -17,7 +17,3 @@ def product_id_change(self):
         res = super().product_id_change()
         self.lot_id = False
         return res
-
-    @api.onchange("product_id")
```

Comment:
Have you replaced by other mechanism ?

Reply:
yes, I've updated the commit with this part.